src/graphql/query/loanChannelPage.graphql

```diff
@@ -13,15 +13,6 @@ query($basketId: String) {
 			}
 		}
 	}
-	shop (basketId: $basketId) {
-		basket {
-			items {
-				values {
-					id
-				}
-			}
-		}
-	}
```

Comment:
This shop data wasn't being used in the only component that used this query, so I removed it.